activerecord/test/models/book_encrypted.rb

```diff
@@ -36,3 +36,10 @@ class EncryptedBookWithUnencryptedDataOptedIn < ActiveRecord::Base
   validates :name, uniqueness: true
   encrypts :name, deterministic: true, support_unencrypted_data: true
 end
+
+class EncryptedBookCustomType < ActiveRecord::Base
+  self.primary_key = "id"
+  self.table_name = "books"
```

Comment:
```suggestion
  self.table_name = "books"
```

Unless tests start to fail we shouldn't need to specify `id` as it can be derived from the table

Reply:
Good catch!